MigFiles/SupportLibraries/ZWaveLib/CommandClasses/DoorLock.cs

```diff
@@ -71,6 +71,12 @@ public static void Get(ZWaveNode node)
         
         public static void Set(ZWaveNode node, Value value)
         {
+            byte bValue;
+            if (value == DoorLock.Value.Secured)
```

Comment:
did you forget to use *bValue* instead of *value* at line 83? can you please also refactor *value* to *lockValue*?